packages/core/utils/.eslintrc.js

```diff
@@ -5,16 +5,16 @@ const RESTRICTED_CONFIG = [
       {
         name: '@parcel/workers',
         message:
-          'Do not import workers inside utils. Instead, create a separate package.'
-      }
-    ]
-  }
+          'Do not import workers inside utils. Instead, create a separate package.',
+      },
+    ],
+  },
 ];
 
 module.exports = {
   extends: '@parcel/eslint-config',
   rules: {
     'no-restricted-imports': RESTRICTED_CONFIG,
-    'no-restricted-modules': RESTRICTED_CONFIG
-  }
+    'no-restricted-modules': RESTRICTED_CONFIG,
+  },
```

Comment:
This PR includes a lot of unrelated stylistic changes probably due to prettier. I think these should be reverted (this isn't the only file)

Reply:
`yarn.lock` is necessary for the integration test to pass because Parcel looks for a lockfile when resolving the root directory of a project. [See here](https://github.com/parcel-bundler/parcel/pull/5304#issuecomment-731567435).

Also, the Prettier changes I agree shouldn't have happened but I'm not sure it's all that important that they're removed. I'll do it if necessary.

Reply:
Honestly I’m surprised that CI doesn’t complain. Is there no code linter?

Reply:
Our `lint`  npm script only checks `./packages/*/*/{src,bin,test}/**/*.{js,json,md}` with prettier. Some time ago we enabled trailing commas and apparently only the files matching the glob were converted.

Reply:
So should these changes be undone or left as-is?

Reply:
You can leave them in.